core/src/main/resources/error/error-classes.json

```diff
@@ -496,7 +496,7 @@
   },
   "ENCODER_NOT_FOUND" : {
     "message" : [
-      "Not found an encoder of the type <typeName> to Spark SQL internal representation. Consider to change the input type to one of supported at https://spark.apache.org/docs/latest/sql-ref-datatypes.html."
+      "Not found an encoder of the type <typeName> to Spark SQL internal representation. Consider to change the input type to one of supported at '<docroot>/sql-ref-datatypes.html'."
```

Comment:
Could you elaborate why do you need quoting by '' around the link.

Reply:
I just mimic the behavior of the existing error messages.

e.g.
```json
  "WRONG_NUM_ARGS" : {
    "message" : [
      "The <functionName> requires <expectedNum> parameters but the actual number is <actualNum>."
    ],
    "subClass" : {
      "WITHOUT_SUGGESTION" : {
        "message" : [
          "Please, refer to 'https://spark.apache.org/docs/latest/sql-ref-functions.html' for a fix."
        ]
      },
```

Do we want to just remove the '' ??